scripts/create_singularities

```diff
@@ -459,7 +459,7 @@ def retry_get(url: str) -> requests.Response:
     while True:
         try:
             return requests.get(url)
-        except (requests.ConnectionError, requests.Timeout) as e:
+        except (requests.ConnectionError, requests.exceptions.HTTPError, requests.Timeout) as e:
```

Comment:
This won't work.  An `HTTPError` is only raised when `raise_for_status()` is called, and it's raised in some situations that you normally wouldn't want to retry on, like 4xx responses.

Reply:
@yarikoptic Which statuses do you want it to retry on?  Just 502?  All 5xx?

Reply:
I would on all 5xx since server might come back to senses

Reply:
Done: 60f7146

Reply:
great, thank you!